aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -13,7 +13,56 @@ uid: fundamentals/servers/kestrel
 
 By [Tom Dykstra](https://github.com/tdykstra), [Chris Ross](https://github.com/Tratcher), and [Stephen Halter](https://twitter.com/halter73)
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
+
+Kestrel is a cross-platform [web server for ASP.NET Core](xref:fundamentals/servers/index). Kestrel is the web server that's included and enabled by default in ASP.NET Core project templates.
+
+Kestrel supports the following scenarios:
+
+* HTTPS
+* [HTTP/2](xref:fundamentals/servers/kestrel/http2) (except on macOS&dagger;)
+* Opaque upgrade used to enable [WebSockets](xref:fundamentals/websockets)
+* Unix sockets for high performance behind Nginx
+
+&dagger;HTTP/2 will be supported on macOS in a future release.
+
+Kestrel is supported on all platforms and versions that .NET Core supports.
+
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/fundamentals/servers/kestrel/samples/5.x) ([how to download](xref:index#how-to-download-a-sample))
+
+## Get started
+
+ASP.NET Core project templates use Kestrel by default. In *Program.cs*, the 
+<xref:Microsoft.Extensions.Hosting.GenericHostBuilderExtensions.ConfigureWebHostDefaults*> method calls <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel*>:
+
+[!code-csharp[](kestrel/samples/5.x/KestrelSample/Program.cs?name=snippet_DefaultBuilder&highlight=8)]
+
+For more information on building the host, see the *Set up a host* and *Default builder settings* sections of <xref:fundamentals/host/generic-host#set-up-a-host>.
+
+## Additional resournces
+
+<a name="kestrel-options"></a>
```

Comment:
I suggest removing the anchors from these links. Instead of referencing these anchors in links elsewhere, link directly to the pertinent section/doc.

Reply:
Except anchors are persistent, and H2 changes break the link. You might argue with a new doc not to use them, but for updates, it's safer to leave them.

Reply:
Exactly. I added these specifically for old external links that could have been referencing H2 sections from previous versions. For external links that use `?view=aspnetcore-3.1` or similar, it won't make a difference. But if someone already linked to something like https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-5.0#kestrel-options, it could help. Maybe not much right now since the root document is now pretty short, but possibly down the road.